services/orbits/area-of-interests/schemas.go

```diff
@@ -0,0 +1,170 @@
+package areas_of_interest
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var validShapeTypes = []string{
+	"POINT", "CIRCLE", "POLYGON",
+}
+
+var areaOfInterestSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"name": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: helper.IsValidName,
+	},
+	"shape": {
+		Type:     schema.TypeList,
+		Required: true,
+		MinItems: 1,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: areaOfInterestShapeSchema,
+		},
+	},
+	"tags": general_objects.KeyValuesSchema,
+	"created_at": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "When it was created",
+	},
+	"created_by": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Who created it",
+	},
+	"last_modified_at": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "When it was last modified",
+	},
+	"last_modified_by": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Who modified it the last",
+	},
+}
+
+var areaOfInterestShapeSchema = map[string]*schema.Schema{
+	"type": {
+		Type:         schema.TypeString,
+		Required:     true, // actually computed, but to create the request, we need it for the validation and parsing
```

Comment:
Can you update the shape with another type? If not you should set `ForceNew: true` like [here](https://github.com/leanspace/terraform-provider-leanspace/blob/main/services/teams/members/schemas.go#L24)